neural_lam/utils.py

```diff
@@ -256,7 +256,7 @@ def fractional_plot_bundle(fraction):
     Get the tueplots bundle, but with figure width as a fraction of
     the page width.
     """
-    bundle = bundles.neurips2023(usetex=True, family="serif")
+    bundle = bundles.neurips2023(usetex=False, family="serif")
```

Comment:
@clechartre explained that using latex introduced issues with some plot annotations. Setting this option to False solved it. (for future references, no action needed)